Cython/Utility/ModuleSetupCode.c

```diff
@@ -329,6 +329,8 @@
 #   endif
 # elif defined(__ICC) || (defined(__INTEL_COMPILER) && !defined(_MSC_VER))
 #   define CYTHON_UNUSED __attribute__ ((__unused__))
+# elif defined(_MSC_VER)
+#   define CYTHON_UNUSED __pragma(warning(suppress:4100))
```

Comment:
Is 4100 specific to function parameters? I think we use `CYTHON_UNUSED` also in a few other places. Does that hurt?

Reply:
Yes it is. I did just find:
```
CYTHON_UNUSED const Py_ssize_t __pyx_nargs = PyTuple_GET_SIZE(__pyx_args);
```
in my code, so it would probably make sense to suppress https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-3-c4101?view=msvc-160 as well -> `warning(suppress: 4100 4101)`

Reply:
4101 is now suppressed a well